backend/src/main/java/gov/cdc/usds/simplereport/api/uploads/FileUploadController.java

```diff
@@ -2,7 +2,6 @@
 
 import static gov.cdc.usds.simplereport.api.Translators.parseUUID;
 import static gov.cdc.usds.simplereport.config.WebConfiguration.CONDITION_AGNOSTIC_RESULT_UPLOAD;
-import static gov.cdc.usds.simplereport.config.WebConfiguration.HIV_RESULT_UPLOAD;
```

Comment:
Can we remove where we define this as well?

Reply:
Nice catch -- updated